app/views/splash/index.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<div class="splash-page d-flex">
+  <div class="home-header-categories header">
+    <h1>Piercel Payments</h1>
+  </div>
+  <div class="user-icon d-flex flex-center">
+    <span style="font-size: 120px;">
+    <i class="fa-solid fa-circle-user"></i></span>
+  </div>
+  <div class="buttons">
+    <%= button_to "Login", new_user_session_path, :method => :get, class:"splash-btns" %>
+    <%= button_to "Sign up", new_user_registration_path, :method => :get, class:"splash-btns splash-signup" %>
+  </div>
+</div>
```

Comment:
- [ ] The styling of your splash page in development seems completely out of place compared to the one deployed, kindly fix that. Whether in development or production, the splash page should follow the original design.

  In development | In production
  :---:|:---:
  ![log-4](https://user-images.githubusercontent.com/36542986/235439658-291c3ce0-acd2-4ee0-a2cf-5d0da365ecfe.jpg) | ![iode](https://user-images.githubusercontent.com/36542986/235439655-d70fe7a2-7d6d-46cd-a66d-18f035703308.jpg)

Reply:
@TedLivist Hi this error happened before I push the code to production. Now I have to pull from the two branches, Then I have the error fixed